frontend/public/components/pod-logs.jsx

```diff
@@ -54,19 +66,21 @@ export class PodLogs extends React.Component {
   }
 
   render() {
-    const {containers, currentKey} = this.state;
+    const {containers, currentKey, initContainers} = this.state;
     const namespace = _.get(this.props.obj, 'metadata.namespace');
     const podName = _.get(this.props.obj, 'metadata.name');
-    const currentContainer = _.get(containers, currentKey);
+    const currentContainer = _.get(containers, currentKey) || _.get(initContainers, currentKey);
+    const currentContainerAlive = _.has(currentContainer.state, 'running');
     const containerDropdown = <ContainerDropdown
-      currentContainer={currentContainer}
+      currentKey={currentKey}
       containers={containers}
+      initContainers={initContainers}
       kind="Container"
```

Comment:
It seems a little strange that `<ContainerDropdown>` API takes a `kind`... I would assume it always has to be `kind="Container"`.

Reply:
Yeah, I thought I got rid of that.